react/src/components/ComputeSessionNodeItems/SessionActionButtons.tsx

```diff
@@ -38,9 +39,7 @@ const isActive = (session: SessionActionButtonsFragment$data) => {
 //   ].includes(session?.status || '');
 // };
 
-const SessionActionButtons: React.FC<SessionActionButtonsProps> = ({
-  sessionFrgmt,
-}) => {
+const SessionActionButtons: React.FC<SessionActionButtonsProps> = (props) => {
```

Comment:
In the webui, Destructuring is the default consistency, is there a reason you changed it?

Reply:
Whether to use destructuring or not is a matter of readability. We can use both approaches at any time, but in most cases, destructuring is better because it's less verbose.

There was no particular reason in this case. ;)